package/MDAnalysis/analysis/rdf.py

```diff
@@ -95,6 +95,10 @@
       :class:`numpy.ndarray` of the centers of the `nbins` histogram
       bins.
 
+       .. deprecated:: 2.0.0
+           This attribute will be removed in 3.0.0.
+           Use :attr:`results.bins` instead.
+
```

Comment:
Should we remove the attribute descriptions given here and put them directly in the docstring of the class? As for example in the GNMAnalysis:

https://github.com/MDAnalysis/mdanalysis/blob/da46e8406b863503004daa06686ba86da0428f39/package/MDAnalysis/analysis/gnm.py#L225-L232

Reply:
I think long term yes, but given the current short deadlines, I'd be inclined to just say let's keep things here and then we can re-format in 2.1.0.

Reply:
@PicoCentauri , if you want to raise or find an issue regarding consistent formatting of analysis classes (which will presumably also aid mdacli in documentation extraction) then please do so and start adding TODOs...